Makefile

```diff
@@ -596,3 +600,15 @@ $(ETCD): $(LOCALBIN)
 			rm $(LOCALBIN)/etcd-temp; \
 		fi \
 	}
+
+.PHONY: yq
+yq: $(YQ) ## Download yq locally if necessary.
+$(YQ): $(LOCALBIN)
+	test -s $(LOCALBIN)/yq || { curl -sfSLo $(YQ) https://github.com/mikefarah/yq/releases/download/$(YQ_VERSION)/yq_linux_$(TARGET_ARCH) && chmod +x $(YQ); }
+
+# modify suites to work on arm64
+# Set cpuScalingMode to SysfsScaling
+# Set targetArchitecture to arm64
+arm_patch_e2e: yq
+	@find neonvm/samples/*yaml tests/e2e -name "*.yaml" | xargs -I{} ./bin/yq eval '(select(.kind == "VirtualMachine") | .spec.cpuScalingMode = "SysfsScaling") // .' -i {}
+	@find neonvm/samples/*yaml tests/e2e -name "*.yaml" | xargs -I{} ./bin/yq eval '(select(.kind == "VirtualMachine") | .spec.targetArchitecture = "arm64") // .' -i {}
```

Comment:
Oof, that's kind of unfortunate, but I see why it's necessary.

`targetArchitecture` I guess will need to be like that until we redefine `nil` to be "no preference", right?

But for `cpuScalingMode`, I wonder if `kuttl` can run only a subset of the tests..? Otherwise I guess that'll be needed until we fully deprecate ACPI CPU hotplug :(

Reply:
With some scripting it is possible to run only a set of tests - kuttl accepts test name to run.
However, I think it is more beneficial for us to cover a wider set of tests on arm until the moment drop ACPI hot plugging, otherwise we just will have no coverage for some of the suites, while the scenario is absolutely possible to happen on staging/production. 

For example, suite for pod restarts. Without that patching the suite would never run on arm. We can skip it, but where is the guarantee that counting logic will work? It has already happened before that counting logic was broken on ARM, but not on AMD.

Reply:
>However, I think it is more beneficial for us to cover a wider set of tests on arm until the moment drop ACPI hot plugging, otherwise we just will have no coverage for some of the suites, while the scenario is absolutely possible to happen on staging/production.

Plus actually, IF I understand correctly, when we drop ACPI hot plug, we will automatically have all those test run on ARM as well, why not to emulate same behavior right now if it provides more safety.

Reply:
Oh, for sure -- I'm not thinking that we should only run the tests that currently will work on ARM, just that if we could run a subset, we can exclude the ones that will never run on ARM (e.g, autoscaling/live migration with QMP scaling mode)

Reply:
I don't think we have any tests that require QMP in particular (I know it is an example!), it is rather a default mode for amd, and default is different for arm. I guess, if we imply it in the code (eg, always use sysfs scaling on arm), there will be no neccessity for the patching at all.